web/styles/app_variables.css

```diff
@@ -180,6 +180,7 @@
     --color-border-dropdown-menu: hsl(0deg 0% 0% / 40%);
     --color-border-personal-menu-avatar: hsl(0deg 0% 0% / 10%);
     --color-background-unread-counter: hsl(105deg 2% 50%);
+    --color-border-add-subscription-button-focus: hsl(0deg 0% 20%);
```

Comment:
This variable seems different from the one used above?

Reply:
Got confused between a variable I created earlier while testing, wondering how it still worked with different variable names especially cause `--color-border-new-style-button-focus` is not defined anywhere.
Fixed it now.